test/test-remote/testutils/logs.ts

```diff
@@ -85,6 +88,7 @@ async function queryLoki(baseUrl: string, queryParams: URLSearchParams) {
   // in the `options` object above and then use
   // `response.body.toString("utf-8")` below but
   // https://github.com/sindresorhus/got/issues/1079
+  // Note: this may throw got.RequestError for request timeout errors.
```

Comment:
(I should have clarified this comment better -- that this depends on `throwHttpErrors: false` and that this is about any transient issue -- next time).